tensorflow/workspace.bzl

```diff
@@ -1126,6 +1126,39 @@ def tf_repositories(path_prefix = "", tf_repo_name = ""):
         ],
     )
 
+    # These are the depedencies for GCS C API Modular Filesystem and will be removed
+    tf_http_archive(
+        name = "com_github_googleapis_google_cloud_cpp",
+        sha256 = "839b2d4dcb36a671734dac6b30ea8c298bbeaafcf7a45ee4a7d7aa5986b16569",
+        strip_prefix = "google-cloud-cpp-1.14.0",
+        patch_file = clean_dep("//third_party:com_github_googleapis_google_cloud_cpp.patch"),
+        urls = [
+            "https://storage.googleapis.com/mirror.tensorflow.org/github.com/googleapis/google-cloud-cpp/archive/v1.14.0.tar.gz",
+            "https://github.com/googleapis/google-cloud-cpp/archive/v1.14.0.tar.gz",
+        ],
+    )
```

Comment:
This dependency already exists and that is causing the build to fail. Do we need this one or is the old one ok?

https://github.com/tensorflow/tensorflow/blob/1145cc3855b216f872cb0596a6cf90c86edc9b88/tensorflow/workspace.bzl#L317-L340